src/Game.js

```diff
@@ -40,217 +36,266 @@ class Game extends Component {
-                </button>
-                <button
-                className="gamepad__control gamepad__control--right"
-                onClick={() => {
-                  const event = { key: 'ArrowRight' };
-                  this.keyAndClickHandler(event);
-                }}>
-                  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="36" height="36"><path fill="none" d="M0 0h24v24H0z"/><path d="M16 12l-6 6V6z" fill="rgba(255,255,255,1)"/></svg>
-                </button>
-                <button
-                className="gamepad__control gamepad__control--left"
-                onClick={() => {
-                  const event = { key: 'ArrowLeft' };
-                  this.keyAndClickHandler(event);
-                }}>
-                  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="36" height="36"><path fill="none" d="M0 0h24v24H0z"/><path d="M8 12l6-6v12z" fill="rgba(255,255,255,1)"/></svg>
-                </button>
-                <button
-                className="gamepad__control gamepad__control--down"
-                onClick={() => {
-                  const event = { key: 'ArrowDown' };
-                  this.keyAndClickHandler(event);
-                }}>
-                  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="36" height="36"><path fill="none" d="M0 0h24v24H0z"/><path d="M12 16l-6-6h12z" fill="rgba(255,255,255,1)"/></svg>
-                </button>
-              </div>
-            </div>
-          )}
-      </section>
-    );
-  }
+                <path fill="none" d="M0 0h24v24H0z" />
+                <path d="M12 8l6 6H6z" fill="rgba(255,255,255,1)" />
+              </svg>
+            </button>
+            <button
+              className="gamepad__control gamepad__control--right"
+              onClick={() => {
+                const event = { key: "ArrowRight" };
+                keyAndClickHandler(event);
+              }}
+            >
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                viewBox="0 0 24 24"
+                width="36"
+                height="36"
+              >
+                <path fill="none" d="M0 0h24v24H0z" />
+                <path d="M16 12l-6 6V6z" fill="rgba(255,255,255,1)" />
+              </svg>
+            </button>
+            <button
+              className="gamepad__control gamepad__control--left"
+              onClick={() => {
+                const event = { key: "ArrowLeft" };
+                keyAndClickHandler(event);
+              }}
+            >
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                viewBox="0 0 24 24"
+                width="36"
+                height="36"
+              >
+                <path fill="none" d="M0 0h24v24H0z" />
+                <path d="M8 12l6-6v12z" fill="rgba(255,255,255,1)" />
```

Comment:
Can you keep the SVG content in one line. It's adding noise by being expanded into 8 lines. Maybe you can do this

```js
const leftArrow = <svg><path="" /></svg>
// similarly for other 3 directions


function Game() {
  return (
    {leftArrow}
    {rightArrow}
    // for other directions
  );
}
```

Reply:
Notice that leftArrow is not a component, it's just some JSX assigned to a variable, that's why it is not named `LeftArrow`

Reply:
What if the count of symbols in one line more than 80 symbols, for example. Is that okay?

Reply:
Yeah that's okay, I have set my prettier print width to 120 because I feel 80 is too low. Another contributor is working on adding a prettierc file to standarise the formatting of this project.

Reply:
I mean that one SVG in one line takes even more than 120 symbols. Should I split by lines if it takes more or not right now? In the future with prettier, they will split automatically, should I care about it right now?

Reply:
Maybe will be better something like this?
```
const upArrow = (
    <svg viewBox="0 0 24 24" width="36" height="36">
      <path fill="none" d="M0 0h24v24H0z" />
      <path d="M12 8l6 6H6z" fill="rgba(255,255,255,1)" />
    </svg>
  );
```

Reply:
this looks great

Reply:
Ok! Thank you 😊